### ISSUE TITLE
how to treat dropout operation when convert tensorflow model

### ISSUE THREAD

**Issue by remarkableJ** [openvinotoolkit/openvino#404]
I dont konw how to treat dropout operation when convert tensorflow model, and cant find any example in openvino model convert guide, please, thx

**Comment by remarkableJ** [comment]
And is there a solution for big data and high CPU usage when inference?

**Comment by maxl16** [comment]
Hello @remarkableJ 
Dropout layer is important for training purposes, however, this is being useless during inference and might increase the inference time. In many cases, such operations can be automatically removed from the resulting Intermediate Representation. 
Dropout layer is not listed as a supported TensorFlow layer here - https://docs.openvinotoolkit.org/latest/_docs_MO_DG_prepare_model_Supported_Frameworks_Layers.html#tensorflow_supported_operations
For the model conversion guide you could check this - https://docs.openvinotoolkit.org/latest/_docs_MO_DG_prepare_model_convert_model_Convert_Model_From_TensorFlow.html
For the big data inference please have a chance to take a look at Intel Analytics Zoo - https://software.intel.com/en-us/ai/analytics-zoo

Best regards, Max.

**Comment by lazarevevgeny** [comment]
Closing as not a bug and answer provided.
